03_products_api/api-material-compositions.md

```diff
@@ -0,0 +1,50 @@
+# Material composition
+
+The material composition is specified in the material composition field "Materialzusammensetzung".
+
+The base for the material composition is the currently valid Textile conventions of the EU. The material composition must be transferred in the following format (please note the structure, blank and separator characters):
+
+![Material composition!](material-composition.gif "Material composition structure")
+
+The list of valid material names can be found [here](material-names.md).
+
+The list of valid material component names can be found [here](material-component-names.md).
+
+Specifically, the following guidelines and syntax rules, among others, apply to the specification of the material composition:
+
+| Rule | Definition  | Examples |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------|
```

Comment:
Diese Markdown Tabelle scheint mir fehlerhaft zu sein. In der gerenderten Ansicht ist die Tabelle nicht zu erkennen, bzw. noch Markdown zu sehen. https://github.com/jofla/retail-api-hub-documentation/blob/main/03_products_api/api-material-compositions.md

Reply:
Vielen Dank fürs Bescheid sagen. Ich hab eine neue Version eingespielt.

Reply:
Jetzt ist gut. :)